test/plug.test.ts

```diff
@@ -771,11 +771,47 @@ describe('The Croct plug', () => {
         expect(evaluate).toHaveBeenCalledWith('user\'s name', {timeout: 5});
     });
 
-    it('should not allow to evaluate expressions if unplugged', () => {
+    it('should log an error when the query evaluation fails', async () => {
+        const logger: Logger = {
+            debug: jest.fn(),
+            info: jest.fn(),
+            warn: jest.fn(),
+            error: jest.fn(),
+        };
+
+        const config: SdkFacadeConfiguration = {
+            appId: APP_ID,
+            logger: logger,
+        };
+
+        const sdkFacade = SdkFacade.init(config);
+
+        const initialize = jest.spyOn(SdkFacade, 'init').mockReturnValue(sdkFacade);
+
+        croct.plug(config);
+
+        expect(initialize).toHaveBeenCalledWith(expect.objectContaining(config));
+
+        const evaluate = jest.spyOn(sdkFacade.evaluator, 'evaluate').mockRejectedValue(new Error('Reason'));
+
+        const query = '"a long query with spaces"';
+
+        const promise = croct.evaluate(query, {timeout: 5});
+
+        await expect(promise).rejects.toThrow('Reason');
+
+        expect(evaluate).toHaveBeenCalledWith(query, {timeout: 5});
+
+        expect(logger.error).toHaveBeenCalledWith(
+            '[Croct] Failed to evaluate query ""a long query with s...": reason',
+        );
+    });
+
+    it('should not allow to evaluate query if unplugged', () => {
         expect(() => croct.evaluate('foo', {timeout: 5})).toThrow('Croct is not plugged in.');
     });
 
-    it('should allow to test expressions', async () => {
+    it('should allow to test query', async () => {
```

Comment:
What about this copy?

```suggestion
    it('should allow to test the query', async () => {
```